students/bbirsoz1/Module_3/mailroom1.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import sys  # imports go at the top of the file
+
+#Mailroom Part 1
+
+donor_db = [("William Gates, III", [653772.32, 12.17]),
+            ("Jeff Bezos", [877.33]),
+            ("Paul Allen", [663.23, 43.87, 1.32]),
+            ("Mark Zuckerberg", [1663.23, 4300.87, 10432.0]),
+            ("Obama", [99999, 3333]),
+            ]
+
```

Comment:
Consider changing your _donor_db_ to a dictionary. It will have a few benefits now and it will also make it easier later in the second course:

```
donor_db = {"William Gates, III": [653772.32, 12.17],
            "Jeff Bezos": [877.33],
            "Paul Allen": [663.23, 43.87, 1.32],
            "Mark Zuckerberg": [1663.23, 4300.87, 10432.0],
            "Obama": [99999, 3333],
} 
```